tests/utils/verification/volumeproperties.go

```diff
@@ -152,6 +155,29 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 	return false
 }
 
+// VerifyDetachedStatusOnNonDefaultDS - For volume on non-default DS,
+// check if the status gets detached within the timeout
+func VerifyDetachedStatusOnNonDefaultDS(name, datastore, hostName, esxName string) bool {
+	log.Printf("Confirming detached status for volume [%s] on datastore[%s]\n", name, datastore)
+
+	//TODO: Need to implement generic polling logic for better reuse
+	const maxAttempt = 60
+	for attempt := 0; attempt < maxAttempt; attempt++ {
+		misc.SleepForSec(2)
+		status := getVolumeStatusHost(name+"@"+datastore, hostName)
```

Comment:
The only difference between this util and VerifyDetachedStatus is the datastore arg which is used to construct a fully qualified name to get volume status from docker host.
If it is worth, please put the common part of these utils into a func which can be called from these utils.

I still think we can use VerifyDetachedStatus, unless we are worried about same volume names on different datastores.